tasks/add-fbc-contribution/add-fbc-contribution.yaml

```diff
@@ -75,6 +75,8 @@ spec:
         request=$(jq -r '.fbc.request // "iib"' ${DATA_FILE})
 
         default_build_timeout_seconds="300"
+        default_request_timeout_seconds=$(params.requestTimeout)
```

Comment:
This of course works, but I think it would be fine to remove the task parameter all together as there is no way to pass a task parameter in our pipelines

Reply:
removed task parameter